nf_core/pipelines/create/template_features.yml

```diff
@@ -52,7 +52,9 @@ ci:
   linting:
     files_exist:
       - ".github/workflows/branch.yml"
-      - ".github/workflows/ci.yml"
+      - ".github/workflows/nf-test.yml"
```

Comment:
Should we also skip the `nf-test.config` files and `tests` directory here?

Reply:
i was thinking about that. but i think one can still want to use nf-test without having it in the ci.